src/routes/view-content/[guideId]/[bibleSection]/+page.svelte

```diff
@@ -215,44 +205,19 @@
         audioPlayerKey = bibleAudioKey(id);
     }
 
-    function showOrDismissBookPassageSelectorPane(show: boolean) {
-        if (show) {
-            bookPassageSelectorPane?.present({ animate: true });
-        } else {
-            bookPassageSelectorPane?.hide();
-        }
-    }
-
-    function showOrDismissBookChapterSelectorPane(show: boolean) {
-        if (show) {
-            bookChapterSelectorPane?.present({ animate: true });
-        } else {
-            bookChapterSelectorPane?.hide();
-        }
-    }
```

Comment:
These have been moved into the pane components instead of handling at the top level.